vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/MainIntegrationTest.java

```diff
@@ -1071,8 +1070,8 @@ public void whenWorkflowRunsAreUnloaded_thenTheyCanBeLoaded() throws IOException
             .statusCode(200)
             .and()
             .extract()
-            .asInputStream();
-    var unload2FileName = MAPPER.readTree(unload2FileQuotedName).toString().replaceAll("\"", "");
+            .jsonPath();
+    var unload2FileName = res.get("filename").toString();
```

Comment:
should unload2FileName come from getting filename from res2?

Reply:
Yes, thanks for catching that! I've updated the test.